shap/explainers/tf_utils.py

```diff
@@ -63,10 +63,12 @@ def _get_model_inputs(model):
         str(type(model)).endswith("keras.engine.training.Model'>") or \
         isinstance(model, tf.keras.Model):
         return model.inputs
-    elif str(type(model)).endswith("tuple'>"):
+    if str(type(model)).endswith("tuple'>"):
         return model[0]
-    else:
-        assert False, str(type(model)) + " is not currently a supported model type!"
+
+    emsg = f"{type(model)} is not currently a supported model type!"
+    raise ValueError(emsg)
```

Comment:
FWIW I slightly prefer the former style on this bit, as the `if / elif / else` style makes it clear that the options are mutually exclusive. They're both functionally equivalent though so happy either way.